### PR TITLE
Adds a happy greeting for my friends the admins who I love!

### PR DESCRIPTION
**Pull request by Aranclanos**
:8ball:

### CODE REVIEW

**Comment by vista-** [comment]
nep-uh-tiz-uh m

**Comment by Niknakflak** [comment]
It's a Trap!!
Now admins will get notified with log-ins and log-offs.  I will no longer be able to sneak up on the new trialmins!

**Comment by WJohn** [comment]
This is adorable :+1:

**Comment by Jalleo** [comment]
Should be checked with admins trainer and above if the greeting should occur. For what niknak said

**Comment by optimumtact** [comment]
This is going to help create a positive and safe space atmosphere for all administrators

:+1:

File: code/modules/admin/admin.dm [hunk 1]
```diff
@@ -805,3 +805,20 @@ var/global/floorIsLava = 0
...
+			"Ah, [key] logged, say hello!",
+			"The server welcomes you [key]!",
+			"Exceptional! [key] joined us, what an honour!",
+			"Another amazing day, because [key] logged, say hi!",
+			"Did anyone see? [key] logged, welcome!",
+			"Monday to sunday, the server warmly welcomes you [key]!",
+			"It's a pleasure to see you [key], greetings!",
+			"Not a bad day if [key] joins, welcome, welcome!",
+			"Surely it's nice to see you [key], have a heartly welcome!")
+			//I really do
```
Comment (Cheridan) [inline code/modules/admin/admin.dm]: i agree with your sentiments completely

**Comment by Fayrik** [comment]
As an Englishman, I cannot stand people being happy to see me.
Please do not merge, or I shall be very offended.

**Comment by Shadowlight213** [comment]
:-1: Unless it says rude things to people with the coder rank.

File: code/modules/admin/admin.dm [hunk 2]
```diff
@@ -805,3 +805,20 @@ var/global/floorIsLava = 0
...
 	return 1
+
+/client/proc/adminGreet()
+	if(ticker && ticker.current_state == GAME_STATE_PLAYING)
+		var/string = pick(
+			"I welcome you to the server [key]!",
+			"Happy to see you [key]!",
+			"Ah, [key] logged, say hello!",
+			"The server welcomes you [key]!",
+			"Exceptional! [key] joined us, what an honour!",
```
Comment (Cheridan) [inline code/modules/admin/admin.dm]: >honour

:angry: :us:

Comment (optimumtact) [inline code/modules/admin/admin.dm]: america banned

Comment (Fayrik) [inline code/modules/admin/admin.dm]: If there's one thing I can't stand more than someone being happy to see me, it's someone talking in an American accent.

:gb: God save the queen. :gb:

**Comment by Razharas** [comment]
Did aran move to canada or something?
Coz thats canadian coding right here

**Pull request merged**